stores/sql/chain.go

```diff
@@ -204,6 +204,28 @@ func FileContractElement(ctx context.Context, tx sql.Tx, fcid types.FileContract
 	}, nil
 }
 
+func IsKnownContract(ctx context.Context, tx sql.Tx, fcid types.FileContractID) (known bool, _ error) {
+	err := tx.QueryRow(ctx, "SELECT 1 FROM contracts WHERE fcid = ?", FileContractID(fcid)).Scan(&known)
```

Comment:
`nit` we usually do `SELECT EXISTS (SELECT 1 ...` to avoid checking `dsql.ErrNoRows` which imo is slightly cleaner